docs/api.md

```diff
@@ -53,18 +53,18 @@ ReactDOM.render(
 
 Connects a React component to a Redux store.
 
-It does not modify the component class passed to it.  
```

Comment:
Whitespace is significant in markdown. By removing these two spaces, you are changing how lines are wrapped.